.python-version

```diff
@@ -0,0 +1 @@
+3.6.0
```

Comment:
This looks like a user-specific file. Should it be unstaged?

Reply:
My apologies! Yes, it should.